### PR TITLE
Add Community section to README

### PR DESCRIPTION
**Pull request by tnozicka**
Now we have ML and Slack (with IRC Bridge).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,3 +90,15 @@ To make it permanent add this line to your `~/.zshrc`.
 ### Documentation
  - [OpenCompose file reference documentation](https://github.com/redhat-developer/opencompose/blob/master/docs/file-reference.md)
  - You can find more in [doc/](https://github.com/redhat-developer/opencompose/tree/master/docs) folder
+
+### Contact
+We always welcome your feedback and thoughts on the project! Come and join us on [Slack](https://opencompose.slack.com) and join our [mailing list](https://groups.google.com/forum/#!forum/opencompose).
+
+#### Slack
+[opencompose.slack.com](https://opencompose.slack.com)
```
Comment (pradeepto) [inline README.md]: Let's not do this please. We should talk on sig-apps on Kubernetes slack instead. If and when time comes, we should get our own channel on the same slack.

**Review by pradeepto — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tnozicka) [reply README.md]: I think we should have both. There should be a channel for discussing the tool, helping users, ... And we should also talk on #sig-apps when we want to engage Kubernetes community; on topics that relate to them. I don't think that spamming #sig-apps with discussing bugs, technological discussions and others would do any good.

Comment (kadel) [reply README.md]: I'm not sure about slack either :-(

1) It is not working as you described ;-) Users need invitation first to sign up to opencompose slack.

2) Isn't ML enough for now?

Comment (tnozicka) [reply README.md]: Ok, Slack sucks. I'll leave only ML and #sig-apps for now.

**Comment by tnozicka** [comment]
@pradeepto @kadel fixed

**Review by pradeepto — APPROVED** [review]

**Comment by pradeepto** [comment]
LGTM

**Pull request merged**